helm-charts/falcon-sensor/values.yaml

```diff
@@ -81,7 +81,33 @@ container:
   enabled: false
 
   # Configure the number of replicas for the mutating webhook backend
-  replicas: 1
+  replicas: 2
+
+  topologySpreadConstraints:
+  - maxSkew: 1
+    topologyKey: kubernetes.io/hostname
+    whenUnsatisfiable: ScheduleAnyway
+    labelSelector:
+      matchLabels:
+        crowdstrike.com/component: crowdstrike-falcon-injector
+
+  affinity:
+    nodeAffinity:
+      # Setting node affinity rule so that replicas will only run on linux nodes
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: kubernetes.io/os
+            operator: In
+            values:
+            - linux
+     # Setting node affinity rule so that replicas will try to run on worker nodes
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 1
+        preference:
+          matchExpressions:
+          - key: node-role.kubernetes.io/master
+            operator: DoesNotExist
```

Comment:
Why was this change made and put into values?

Reply:
@redhatrises I was doing this change when I tried to add podAntiAffinity to affinity statements and provide user to override the default affinity values.But since I am using pod topology spread and no longer updating affinity statements, I just left this as it is in values.yaml.Is there any reason to not put in values.yaml? I can revert back this change if you think there is no reason to allow affinity statements.